### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by itajaja**

### CODE REVIEW

**Comment by taion** [comment]
we should probably disable the "push" build on travis and just have the "PR" build

**Comment by itajaja** [comment]
> moto is the best

it really is! [this](https://github.com/localstack/localstack) is also really cool

> we should probably disable the "push" build on travis and just have the "PR" build

why's that?

**Comment by taion** [comment]
we don't get anything out of the "branch" build. it's just building the same thing twice

File: tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import os
+
+os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```
Comment (taion) [inline tests/__init__.py:3]: oof, can't set this in travis?

Comment (itajaja) [reply tests/__init__.py:3]: and then have local tests not pass? don't see the point

**Pull request merged**